libmscore/harmony.cpp

```diff
@@ -1452,8 +1452,8 @@ void Harmony::drawEditMode(QPainter* p, EditData& ed)
 
 TextSegment::TextSegment(const QString& s, const QFont& f, qreal x, qreal y)
       {
-      QPointF offset(0.0, 0.0);
-      set(s, f, x, y, offset);
+      QPointF off(0.0, 0.0);
+      set(s, f, x, y, off);
       select = false;
```

Comment:
What's going on here? Creating a zero point variable to send it to a function as a parameter? I would send QPointF() as a parameter without creating dummy variable. If the parameter is a reference, I would call the variable `dummyPoint`

Reply:
Indeed, I just fixed the warning without further thinking